<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>panini</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QLabel" name="projLabel" >
    <property name="geometry" >
     <rect>
      <x>150</x>
      <y>150</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text" >
     <string> proj: cube</string>
    </property>
   </widget>
   <widget class="QLabel" name="hfovLabel" >
    <property name="geometry" >
     <rect>
      <x>230</x>
      <y>150</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text" >
     <string>hfov 360.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="vfovLabel" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>300</x>
      <y>150</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text" >
     <string>vfov 180.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="surfaceLabel" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>370</x>
      <y>150</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text" >
     <string>panocylinder</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_View" >
    <property name="title" >
     <string>&amp;View</string>
    </property>
    <addaction name="actionPan_Right" />
    <addaction name="actionPan_Left" />
    <addaction name="actionTilt_Up" />
    <addaction name="actionTilt_Down" />
    <addaction name="actionRoll_Right" />
    <addaction name="actionRoll_Left" />
    <addaction name="separator" />
    <addaction name="actionZoom_In" />
    <addaction name="actionZoom_Out" />
    <addaction name="actionEye_In" />
    <addaction name="actionEye_Out" />
    <addaction name="separator" />
    <addaction name="actionHFovUp" />
    <addaction name="actionHFovDn" />
    <addaction name="actionVFovUp" />
    <addaction name="actionVFovDn" />
    <addaction name="separator" />
    <addaction name="actionSave_as" />
   </widget>
   <widget class="QMenu" name="menuLoad" >
    <property name="title" >
     <string>&amp;Picture</string>
    </property>
    <addaction name="actionQTVR" />
    <addaction name="actionRectilinear" />
    <addaction name="actionFisheye" />
    <addaction name="actionSpherical" />
    <addaction name="actionCylindrical" />
    <addaction name="actionEquirectangular" />
    <addaction name="actionStereographic" />
    <addaction name="actionMercator" />
    <addaction name="actionCube_faces" />
    <addaction name="actionPT_script" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionAbout_pvQt" />
    <addaction name="actionMouse_modes" />
   </widget>
   <widget class="QMenu" name="menuPresets" >
    <property name="title" >
     <string>Presets</string>
    </property>
    <addaction name="actionSuper_wide" />
    <addaction name="actionFull_frame" />
    <addaction name="action_Home" />
    <addaction name="actionReset" />
    <addaction name="action90_deg_CCW" />
    <addaction name="action90_deg_CW" />
    <addaction name="actionNext_iProj" />
    <addaction name="actionPrev_iProj" />
    <addaction name="separator" />
    <addaction name="actionPanosphere" />
    <addaction name="actionPanocylinder" />
   </widget>
   <addaction name="menuLoad" />
   <addaction name="menu_View" />
   <addaction name="menuPresets" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="action_Home" >
   <property name="text" >
    <string>Home Y,P,R</string>
   </property>
   <property name="shortcut" >
    <string>Home</string>
   </property>
  </action>
  <action name="actionZoom_In" >
   <property name="text" >
    <string>Zoom &amp;in</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Up</string>
   </property>
  </action>
  <action name="actionZoom_Out" >
   <property name="text" >
    <string>Zoom &amp;out</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Down</string>
   </property>
  </action>
  <action name="actionPan_Left" >
   <property name="text" >
    <string>Yaw &amp;left</string>
   </property>
   <property name="shortcut" >
    <string>Left</string>
   </property>
  </action>
  <action name="actionPan_Right" >
   <property name="text" >
    <string>Yaw &amp;right</string>
   </property>
   <property name="shortcut" >
    <string>Right</string>
   </property>
  </action>
  <action name="actionTilt_Up" >
   <property name="text" >
    <string>Pitch &amp;up</string>
   </property>
   <property name="shortcut" >
    <string>Up</string>
   </property>
  </action>
  <action name="actionTilt_Down" >
   <property name="text" >
    <string>Pitch &amp;down</string>
   </property>
   <property name="shortcut" >
    <string>Down</string>
   </property>
  </action>
  <action name="actionEye_In" >
   <property name="text" >
    <string>Eye &amp;near</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="actionEye_Out" >
   <property name="text" >
    <string>Eye &amp;far</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="actionFullFrame" >
   <property name="text" >
    <string>Fullframe</string>
   </property>
   <property name="shortcut" >
    <string>PgDown</string>
   </property>
  </action>
  <action name="action_SuperFish" >
   <property name="text" >
    <string>SuperFish</string>
   </property>
   <property name="shortcut" >
    <string>PgUp</string>
   </property>
  </action>
  <action name="actionReset" >
   <property name="text" >
    <string>Reset view</string>
   </property>
   <property name="shortcut" >
    <string>End</string>
   </property>
  </action>
  <action name="actionRectilinear" >
   <property name="text" >
    <string>rectilinear</string>
   </property>
  </action>
  <action name="actionFisheye" >
   <property name="text" >
    <string>fisheye</string>
   </property>
  </action>
  <action name="actionCylindrical" >
   <property name="text" >
    <string>cylindrical</string>
   </property>
  </action>
  <action name="actionEquirectangular" >
   <property name="text" >
    <string>equirectangular</string>
   </property>
  </action>
  <action name="actionCube_faces" >
   <property name="text" >
    <string>cube faces</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>Quit</string>
   </property>
  </action>
  <action name="actionQTVR" >
   <property name="text" >
    <string>QTVR</string>
   </property>
  </action>
  <action name="actionPT_script" >
   <property name="text" >
    <string>PT script</string>
   </property>
  </action>
  <action name="actionAbout_pvQt" >
   <property name="text" >
    <string>About...</string>
   </property>
  </action>
  <action name="actionFull_frame" >
   <property name="text" >
    <string>Full frame</string>
   </property>
   <property name="shortcut" >
    <string>F</string>
   </property>
  </action>
  <action name="actionSuper_wide" >
   <property name="text" >
    <string>Super wide</string>
   </property>
   <property name="shortcut" >
    <string>S</string>
   </property>
  </action>
  <action name="action90_deg_CW" >
   <property name="text" >
    <string>Turn right</string>
   </property>
   <property name="shortcut" >
    <string>R</string>
   </property>
  </action>
  <action name="action90_deg_CCW" >
   <property name="text" >
    <string>Turn left</string>
   </property>
   <property name="shortcut" >
    <string>L</string>
   </property>
  </action>
  <action name="actionSpherical" >
   <property name="text" >
    <string>spherical</string>
   </property>
  </action>
  <action name="actionStereographic" >
   <property name="text" >
    <string>stereographic</string>
   </property>
  </action>
  <action name="actionMercator" >
   <property name="text" >
    <string>mercator</string>
   </property>
  </action>
  <action name="actionHFovUp" >
   <property name="text" >
    <string>hFov+</string>
   </property>
   <property name="shortcut" >
    <string>Shift+Right</string>
   </property>
  </action>
  <action name="actionHFovDn" >
   <property name="text" >
    <string>hFov-</string>
   </property>
   <property name="shortcut" >
    <string>Shift+Left</string>
   </property>
  </action>
  <action name="actionVFovUp" >
   <property name="text" >
    <string>vFov+</string>
   </property>
   <property name="shortcut" >
    <string>Shift+Up</string>
   </property>
  </action>
  <action name="actionVFovDn" >
   <property name="text" >
    <string>vFov-</string>
   </property>
   <property name="shortcut" >
    <string>Shift+Down</string>
   </property>
  </action>
  <action name="actionSave_as" >
   <property name="text" >
    <string>Save as...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNext_iProj" >
   <property name="text" >
    <string>Next iProj</string>
   </property>
   <property name="shortcut" >
    <string>Shift+PgUp</string>
   </property>
  </action>
  <action name="actionPrev_iProj" >
   <property name="text" >
    <string>Prev iProj</string>
   </property>
   <property name="shortcut" >
    <string>Shift+PgDown</string>
   </property>
  </action>
  <action name="actionMouse_modes" >
   <property name="text" >
    <string>Mouse modes...</string>
   </property>
  </action>
  <action name="actionRoll_Right" >
   <property name="text" >
    <string>Roll CW</string>
   </property>
   <property name="shortcut" >
    <string>Alt+Right</string>
   </property>
  </action>
  <action name="actionRoll_Left" >
   <property name="text" >
    <string>Roll CCW</string>
   </property>
   <property name="shortcut" >
    <string>Alt+Left</string>
   </property>
  </action>
  <action name="actionPanosphere" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>panosphere</string>
   </property>
   <property name="toolTip" >
    <string>project on sphere</string>
   </property>
  </action>
  <action name="actionPanocylinder" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>panocylinder</string>
   </property>
   <property name="toolTip" >
    <string>project on cylinder</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
